addons/mail/static/tests/tours/user_modify_own_profile_tour.js

```diff
@@ -19,7 +20,13 @@ registry.category("web_tour.tours").add("mail/static/tests/tours/user_modify_own
         {
             content: "Update the email address",
             trigger: 'div[name="email"] input',
-            run: "edit updatedemail@example.com",
+            async run() {
+                const input = document.querySelector("div[name='email'] input");
+                input.value = "updatedemail@example.com";
+                input.dispatchEvent(new Event("input", { bubbles: true, cancelable: false }));
+                input.dispatchEvent(new Event("change", { bubbles: true, cancelable: false }));
```

Comment:
Can't we use `insertText` if we import `contains` utils?

Reply:
done